cclib/parser/orcaparser.py

```diff
@@ -76,6 +76,14 @@ def after_parsing(self):
                     break
                 self.scfenergies[i] += dispersionenergy
 
+
+    def skip_until_no_match_line(self, inputfile, regex):
```

Comment:
Can this get a docstring and be placed after https://github.com/cclib/cclib/blob/204b85c3c3dfe00208205fac54f737dca9b8562e/cclib/parser/logfileparser.py#L475

That way it can be used in other parsers as well.